src/bundle.c

```diff
@@ -159,6 +159,20 @@ static gboolean casync_make_arch(const gchar *idxpath, const gchar *contentpath,
 		g_ptr_array_add(iargs, g_strdup("--store"));
 		g_ptr_array_add(iargs, g_strdup(store));
 	}
+	if (r_context()->casync_args != NULL) {
+		g_auto(GStrv) casync_argvp = NULL;
+		res = g_shell_parse_argv(r_context()->casync_args, NULL, &casync_argvp, &ierror);
```

Comment:
Now that `res` is already set to TRUE here, it is required to explicitly set `res = FALSE` in the error path of the `r_subprocess_newv()` call below.

Reply:
good catch ; resolved !